tools/BuildAssets/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -394,14 +411,15 @@ function Start-AutoRestCodeGenerationWithLocalConfig {
         [string] $Namespace,
 
         [Parameter(Mandatory = $false)]
-        [string] $ConfigFileTag
+        [ValidateSet("single-api", "multi-api")]
+        [string] $ApiType
```

Comment:
@dsgouda how come this has the validate set, but line 136 does not?

Reply:
This function is not exported and the validation of parameters happens in the functions that actually call this function,
Validation here would be redundant